pipeline/terraform/main.tf

```diff
@@ -66,11 +66,11 @@ module "catalogue_pipeline_2021-03-09" {
 }
 
 
-module "catalogue_pipeline_2021-03-05" {
+module "catalogue_pipeline_2021-03-15" {
   source = "./stack"
 
-  pipeline_date = "2021-03-05"
-  release_label = "prod"
+  pipeline_date = "2021-03-15"
+  release_label = "stage"
```

Comment:
is reading from this index?perhaps a stupid question, but why is this pipeline "stage" if the API is reading from this index?